server/plugin.go

```diff
@@ -72,6 +73,15 @@ type Plugin struct {
 
 	WebhookHandler webhook.Webhook
 	GitlabClient   gitlab.Gitlab
+	// gitlabPermalinkRegex is used to parse gitlab permalinks in post messages.
+	gitlabPermalinkRegex *regexp.Regexp
+}
+
+// NewPlugin returns an instance of a Plugin.
+func NewPlugin() *Plugin {
+	return &Plugin{
```

Comment:
0/5 a static `var gitlabPermalinkRegex = regexp.Must...` would be a touch more idiomatic, since it never changes and does not need to be per-plugin?

Reply:
Done